test.js

```diff
@@ -100,24 +101,35 @@ test('support sync function with plan', function(t) {
   t.equal(result, 42);
 });
 
-test('support arrows with plan', t => {
```

Comment:
The test above (line 97) is already a sync test with a plan call. As far as I can see exactly the same test as the one that was deleted.